src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -31,6 +31,10 @@
     private Date reportEndDate;
     private String policy;
     private String expenseType;
+    private KualiDecimal cashAdvanceAmount;
+    private Date cashAdvanceIssueDate;
+    private String cashAdvanceCaKey;
```

Comment:
I would suggest renaming this to just "cashAdvanceKey" for simplicity, along with the other places in this PR that reference this or have a similar naming pattern.